config/deploy.rb

```diff
@@ -44,6 +44,9 @@
   'config/property_permissions.yml',
   'config/redis.yml',
   'config/resque-pool.yml',
+  'config/resque-web.rb',
+  'config/initializers/resque_config.rb',
+  'config/initializers/resque_admin.rb',
```

Comment:
OK. I guess it doesn't seem like a good choice to me because that's more files to have to manage on the server side. If they don't change that much, then I don't see an incentive to manage them elsewhere. If you deploy the application to a new server, you now have three more files that you have to manage.